tests/Types/IDEntityCantAutodetectTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace AvtoDev\IDEntity\Tests\Types;
+
+use AvtoDev\IDEntity\Tests\AbstractTestCase;
+use AvtoDev\IDEntity\Tests\Mocks\IDEntityCantAutodetectMock;
+
+/**
+ * Class IDEntityBodyTest.
+ */
+class IDEntityCantAutodetectTest extends AbstractTestCase
+{
+    /**
+     * Тестируем, что сущность не может быть автоматически определена.
+     */
+    public function testCanAutodetectMethod()
+    {
+        $instance = new IDEntityCantAutodetectMock('');
+
```

Comment:
Dosen't see test with set of types.

For example:
- 1 and 2 can be autodetected, and 3 - not. So, we must assert that autodecect returns object with 2nd type.
- 1, 2 and 4 can be autodetected, and 3 - not. Assert that autodecect returns object with 4th type.

Reply:
That is unnecessary, because similar behavior was tested in
https://github.com/avto-dev/identity-laravel/blob/master/tests/IDEntityTest.php#L205 - we don't return "cant_autodetect_type"
In the same in your examples has the meaning order of entities in supported types, but it wasn't tested too:
https://github.com/avto-dev/identity-laravel/blob/master/tests/IDEntityTest.php#L190

Reply:
This test is required anyway